frontend/apps/marketing/src/components/contentful/doubleTheDonation/DoubleTheDonation.tsx

```diff
@@ -0,0 +1,70 @@
+'use client';
+
+import React, {useEffect, useState} from 'react';
+
+import {BodyOneText} from '@code-dot-org/component-library/typography';
+
+import {DOUBLE_THE_DONATION_PUBLIC_KEY} from '@/config/doubleTheDonation';
+
+declare global {
+  interface Window {
+    DDCONF?: {API_KEY: string};
+  }
+}
+
+const DoubleTheDonationSearch: React.FC = () => {
+  const [isClient, setIsClient] = useState(false);
+
+  useEffect(() => {
+    setIsClient(true);
+
+    if (typeof window !== 'undefined') {
+      window.DDCONF = {
+        API_KEY: DOUBLE_THE_DONATION_PUBLIC_KEY,
+      };
+
+      const script = document.createElement('script');
+      script.src = 'https://doublethedonation.com/api/js/ddplugin.js';
+      script.async = true;
+      document.body.appendChild(script);
+
+      return () => {
+        if (document.body.contains(script)) {
+          document.body.removeChild(script);
+        }
+      };
+    }
+  }, []);
+
+  if (!isClient) {
```

Comment:
I believe the whole `isClient` state can be removed with `typeof window === 'undefined'`

```suggestion
  if (typeof window === 'undefined') {
```

Reply:
You're right, removed!